packages/web/webpack.config.js

```diff
@@ -37,6 +37,9 @@ module.exports = {
   devtool: 'cheap-eval-source-map',
   resolve: {
     extensions: ['.js', '.jsx', '.ts', '.tsx'],
+    alias: {
+      'react-dom': '@hot-loader/react-dom'
```

Comment:
is this needed?

Reply:
yes, to solve this issue: https://github.com/gaearon/react-hot-loader/issues/1227#issuecomment-482139583